redis-worker/src/index.ts

```diff
@@ -0,0 +1,56 @@
+import axios from 'axios';
+import { createClient } from 'redis';
+import config from './config';
+
+export async function main() {
+  const redisClient = createClient({ 
+    url: `redis://${config.redis.host}:${config.redis.port}`
+   });
+
+  try {
+    await redisClient.connect();
+  } catch (error) {
+    console.error(`Error connecting to redis with url redis://${config.redis.host}:${config.redis.port}:`, error);
+  }
+
+  try {
+    while (true) {
+      const data = await redisClient.lRange(config.redis.key!, 0, config.batchSize - 1);
+      if (data.length === 0) {
+        console.log('No data in queue, waiting...');
+        await new Promise(resolve => setTimeout(resolve, 1000 * 10)); // 10 seconds
+        continue;
+      }
+
+      await updatePostgrest(data);
+      console.log('Data processed:', data);
+
+      const removedCount = await redisClient.lTrim(config.redis.key!, config.batchSize, -1);
+      console.log(`Updated ${data.length} items to PostgREST, removed ${removedCount} from queue.`);
```

Comment:
its a small thing but ltrim returns OK not the count so this logs removed OK from queue

Reply:
Yeah, I forgot to edit that log to remove that part. I will do that.